1-exercises/A-reading-list/script.js

```diff
@@ -1,7 +1,33 @@
 function readingList(books) {
   // Write your code here...
+  let getContent = document.querySelector("#content");
+  let ulElement = document.createElement("ul");
```

Comment:
So here is an example `ulElement `.  Can you think of a better name? How about `booklistElement`.  It's something to think about.